CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#4779](https://github.com/thanos-io/thanos/pull/4779) Examples: Fix the interactive test for MacOS users.
 - [#4792](https://github.com/thanos-io/thanos/pull/4792) Store: Fix data race in BucketedBytes pool.
 - [#4769](https://github.com/thanos-io/thanos/pull/4769) Query-frontend+api: add "X-Request-ID" field and other fields to start call log.
+- [#4918](https://github.com/thanos-io/thanos/pull/4918) Tracing: Fixing force tracing with Jaeger.
```

Comment:
Hmm I'm not sure this needs to be added to the changelog

Reply:
Sorry, @squat , could you elaborate what you mean?

Reply:
sure :) normally we reserve the changelog for user-facing / user-impacting changes. if this is change is an implementation detail that doesn't affect users directly, then we can omit the new entry from the changelog

Reply:
Well... users like me before were not able to use forcing tracing with header `X-Thanos-Force-Tracing` as described in documentation https://thanos.io/tip/thanos/tracing.md/#forcing-sampling, after that change they are able to do so. Also https://github.com/thanos-io/thanos/pull/4018 had CHANGELOG entry. But I'm only guest here, if it's need to be gone - sure, let me know. :)

Reply:
Sorry, /cc @squat ☝️

Reply:
cool, sounds reasonable then :))